source/DSCResources/DSC_Computer/DSC_Computer.psm1

```
@@ -91,7 +96,11 @@ function Get-TargetResource
         [Parameter()]
         [ValidateSet('AccountCreate', 'Win9XUpgrade', 'UnsecuredJoin', 'PasswordPass', 'JoinWithNewName', 'JoinReadOnly', 'InstallInvoke')]
         [System.String[]]
-        $Options
+        $Options,
+
+        [Parameter()]
+        [System.Boolean]
+        $DeleteExistingComputerAccount = $true
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'DeleteExistingComputerAccount' has been declared but not used.

(PSReviewUnusedParameter)